pages/assets/table-style.js

```diff
@@ -0,0 +1,77 @@
+/* stylelint-disable */
+import { css } from 'lit-element';
+
+export const customTableStyles = css`
+	table {
+		border: 1px solid #dfe6ef;
+		border-radius: 0.3rem;
+		border-spacing: 0;
+		width: 100%;
+	}
+
+	tr > td,
+	tr > th {
+		font-weight: inherit;
+		padding: 12px 6px 12px 6px;
+		text-align: left;
+		vertical-align: middle;
+	}
+
+	/* table > tbody > tr:nth-child(1) > th */
+	tbody:first-child > tr > th {
+		background-color: var(--d2l-color-sylvite);
+		font-weight: bold;
+	}
+
+	/* override background color or headers that live within rows */
+	tbody > tr > th {
+		background-color: white;
+	}
+
+	/* padding for extra space on the left table edges */
+	tbody > tr > td:first-child,
+	tbody > tr > th,
+	thead > tr > th:first-child {
```

Comment:
I think you can just leave the tag name off:

```suggestion
	tbody > tr > *:first-child {
```